src/phoenix/server/api/mutations/chat_mutations.py

```diff
@@ -79,21 +96,188 @@ class ChatCompletionMutationPayload:
     error_message: Optional[str]
 
 
+@strawberry.type
+class ChatCompletionMutationError:
+    message: str
```

Comment:
theres an error_message in the payload, do we need both? I don't have a strong opinion on multiple optional fields or a union but maybe @anticorrelator or @axiomofjoy did?

Reply:
This new one is for things like `TemplateFormatterError` which happens before the span is created, whereas the existing one is for the error recorded inside the span.

Reply:
ahhi keep forgetting about the span so the other one is there for the case where
1. we started some work and started a span
2. something went wrong
3. we return span + error
This one is for, no real work even started, got an error first?

Reply:
Yea, for the subscription version currently, it just stops all work and returns an error for the entire dataset. We could also do the same here that's preferred.